spec.html

```diff
@@ -37,8 +37,75 @@
 <emu-grammar>UnicodePropertyValueExpression :: LoneUnicodePropertyNameOrValue</emu-grammar>
 <ul>
   <li>
-    It is a Syntax Error if the List of Unicode code points that is SourceText of |LoneUnicodePropertyNameOrValue| is not identical to a List of Unicode code points that is a Unicode general category or general category alias listed in the &ldquo;Property value and aliases&rdquo; column of <emu-xref href="#table-unicode-general-category-values"></emu-xref>, nor a binary property or binary property alias listed in the &ldquo;Property name and aliases&rdquo; column of <emu-xref href="#table-binary-unicode-properties"></emu-xref><ins>, nor a sequence property listed in the &ldquo;Property name and aliases&rdquo; column of <emu-xref href="#table-unicode-sequence-properties"></emu-xref></ins>.
+    It is a Syntax Error if the List of Unicode code points that is SourceText of |LoneUnicodePropertyNameOrValue| is not identical to a List of Unicode code points that is a Unicode general category or general category alias listed in the &ldquo;Property value and aliases&rdquo; column of <a href="https://tc39.github.io/ecma262/#table-unicode-general-category-values">Table 53</a>, nor a binary property or binary property alias listed in the &ldquo;Property name and aliases&rdquo; column of <a href="https://tc39.github.io/ecma262/#table-binary-unicode-properties">Table 52</a><ins>, nor a sequence property or sequence property alias listed in the &ldquo;Property name and aliases&rdquo; column of <emu-xref href="#table-unicode-sequence-properties"></emu-xref></ins>.
+  </li>
+  <li>
+    <ins>It is a Syntax Error if |LoneUnicodePropertyNameOrValue| occurs within a |CharacterClass| production and the List of Unicode code points that is SourceText of |LoneUnicodePropertyNameOrValue| is identical to a List of Unicode code points that is a Unicode sequence property or sequence property alias listed in the &ldquo;Property name and aliases&rdquo; column of <emu-xref href="#table-unicode-sequence-properties"></emu-xref>.</ins>
+  </li>
+  <li>
+    <ins>It is a Syntax Error if |LoneUnicodePropertyNameOrValue| occurs within a <emu-grammar>CharacterClassEscape :: `P{` UnicodePropertyValueExpression `}`</emu-grammar> production and the List of Unicode code points that is SourceText of |LoneUnicodePropertyNameOrValue| is identical to a List of Unicode code points that is a Unicode sequence property or sequence property alias listed in the &ldquo;Property name and aliases&rdquo; column of <emu-xref href="#table-unicode-sequence-properties"></emu-xref>.</ins>
```

Comment:
Is this the right way of banning this within `\P{…}`?

[Do we even want to ban this?](https://github.com/mathiasbynens/proposal-regexp-unicode-sequence-properties/issues/6)